pyanaconda/modules/storage/devicetree/viewer.py

```diff
@@ -501,10 +503,15 @@ def get_existing_systems(self):
         os_list = list(map(self._get_os_data, self.storage.roots))
 
         # Append windows systems if windows partition types are present
-        windows_data = self._get_windows_data()
+        windows_data = self._get_other_os_data("Windows")
```

Comment:
Can't we make the "Windows" and "Mac OS" constants. This is prone to typos when the string have to be the same in multiple functions.